mne/event.py

```diff
@@ -187,8 +188,10 @@ def _read_events_fif(fid, tree):
     event_list = event_list.reshape(len(event_list) // 3, 3)
     return event_list, mappings
 
-
-def read_events(filename, include=None, exclude=None, mask=0):
+@deprecated("The default setting for the argument 'mask_type' will change "
+            "from None to 'and' in v0.14.")
```

Comment:
This is not how we deprecate a parameter. The warning should only be emitted if `mask_type=None`, what you have here will *always* give a warning, even if a user explicitly does `mask_type=whatever`. See how we do it elsewhere, `git grep DeprecationWarning`.